storage/google/cloud/storage/blob.py

```diff
@@ -1457,7 +1457,7 @@ def rewrite(self, source, token=None, client=None):
 
         return api_response['rewriteToken'], rewritten, size
 
-    def update_storage_class(self, new_class, client=None):
+    def update_storage_class(self, new_class, token=None, client=None):
```

Comment:
This would be a breaking change as anyone that called `update_storage_class(new_class, opt_client)` is now assigning opt_client to token. Though this was done to match the parameter order of rewrite. If we invert the two params here though we can avoid breaking this API.

There are other ways we can mitigate this later if we wanted to fix this to match though.

Reply:
Sure, I'm happy to swap them.